info/v2/conversion.go

```diff
@@ -96,7 +96,7 @@ func MachineStatsFromV1(cont *v1.ContainerInfo) []MachineStats {
 	return stats
 }
 
-func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []*v1.ContainerStats) []*ContainerStats {
+func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []*v1.ContainerStats, firstCall bool) []*ContainerStats {
```

Comment:
I'm not really thrilled about doing any of those, let me think about it.  In general I wish this was something log frameworks were a little smarter about.